Allows gcc 4.2.1 & gcc 4.2.4 to accept --with-abi=aapcs-linux at configure
stage.

Taken from buildroot/toolchain/gcc/4.3.x/

Index: buildroot/toolchain/gcc/4.2.1/999-4.2.1-armeabi-aapcs-linux.patch
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ buildroot/toolchain/gcc/4.2.1/999-4.2.1-armeabi-aapcs-linux.patch	2012-02-21 16:44:57.000000000 +0100
@@ -0,0 +1,18 @@
+\\\\ followup of PR34205; trying to use aapcs-linux
+\\ gcc/ChangeLog
+\\
+\\ 2008-09-18  Bernhard Reutner-Fischer  <aldot@>
+\\
+\\ * config.gcc (arm*-*-*): Add aapcs-linux to supported ABIs.
+\\
+--- gcc-4.2.1.orig/gcc/config.gcc	2008-09-18 20:33:55.000000000 +0200
++++ gcc-4.2.1/gcc/config.gcc	2008-09-18 21:38:52.000000000 +0200
+@@ -2714,7 +2714,7 @@
+ 
+ 		case "$with_abi" in
+ 		"" \
+-		| apcs-gnu | atpcs | aapcs | iwmmxt )
++		| apcs-gnu | atpcs | aapcs | iwmmxt | aapcs-linux )
+ 			#OK
+ 			;;
+ 		*)
Index: buildroot/toolchain/gcc/4.2.4/999-4.2.4-armeabi-aapcs-linux.patch
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ buildroot/toolchain/gcc/4.2.4/999-4.2.4-armeabi-aapcs-linux.patch	2012-02-21 16:44:57.000000000 +0100
@@ -0,0 +1,18 @@
+\\\\ followup of PR34205; trying to use aapcs-linux
+\\ gcc/ChangeLog
+\\
+\\ 2008-09-18  Bernhard Reutner-Fischer  <aldot@>
+\\
+\\ * config.gcc (arm*-*-*): Add aapcs-linux to supported ABIs.
+\\
+--- gcc-4.2.4.orig/gcc/config.gcc	2008-09-18 20:33:55.000000000 +0200
++++ gcc-4.2.4/gcc/config.gcc	2008-09-18 21:38:52.000000000 +0200
+@@ -2714,7 +2714,7 @@
+ 
+ 		case "$with_abi" in
+ 		"" \
+-		| apcs-gnu | atpcs | aapcs | iwmmxt )
++		| apcs-gnu | atpcs | aapcs | iwmmxt | aapcs-linux )
+ 			#OK
+ 			;;
+ 		*)
